ChurchToolsApi.py

```diff
@@ -477,18 +477,35 @@ def get_songs_with_tag(self, song_tag_id: int):
 
         return result
 
-    def get_events(self, event_id=None):
-        """
-        Retrieve list of events from ChurchTools
-        :return: a list of events
-        #TODO 2 There are params available which are not implemented yet!
+    def get_events(self, from_: str = '', to: str = '', canceled: bool = False, direction: str = 'forward',
```

Comment:
style question - i've started to use kwargs instead of defaulting params for optional params - which way do you prefer?

Reply:
You didn't happen to write test methods for the new params did you? I would like to add them before merging into master.

Reply:
OK. kwargs is a good way.
I will also use it.